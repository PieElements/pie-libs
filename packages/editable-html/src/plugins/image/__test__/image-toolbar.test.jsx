import React from 'react';
import Toolbar from '../image-toolbar';
import renderer from 'react-test-renderer';
import {Data} from 'slate';

it('renders correctly', () => {

  const node = { 
    data: Data.create({
    resizePercent: 100
  })};

  const classes = {holder:'holder'};

  const tree = renderer
    .create(<Toolbar node={node} classes={classes} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});