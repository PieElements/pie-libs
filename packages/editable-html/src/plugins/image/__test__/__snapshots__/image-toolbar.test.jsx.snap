// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="holder"
>
  <span
    className="button-0-1"
    onMouseDown={[Function]}
  >
    25%
  </span>
  <span
    className="button-0-1"
    onMouseDown={[Function]}
  >
    50%
  </span>
  <span
    className="button-0-1"
    onMouseDown={[Function]}
  >
    75%
  </span>
  <span
    className="button-0-1 active-0-2"
    onMouseDown={[Function]}
  >
    100%
  </span>
</div>
`;
